src/scripts/browser.js

```diff
@@ -14,6 +14,19 @@ function isTv() {
         return true;
     }
 
+    if (isWeb0s()) {
+        return true;
+    }
+
+    return false;
+}
+
+function isWeb0s() {
+    const userAgent = navigator.userAgent.toLowerCase();
+    if (userAgent.indexOf('netcast') !== -1) {
```

Comment:
just for a better look
```suggestion
    const userAgent = navigator.userAgent.toLowerCase();

    if (userAgent.indexOf('netcast') !== -1) {
```